README.md

```diff
@@ -6,7 +6,7 @@ To build and install the snap:
 ```
  $ snapcraft
  ...
- $ snap install --devmode --dangerous plexmediaserver_*_amd64.snap
+ $ sudo snap install --devmode --force-dangerous plexmediaserver_*_amd64.snap
```

Comment:
I'm running 16.04 which has 2.14.2 which is what most people are going to be using for now. I will add another line for snapd 2.14 vs snapd 2.15+